src/service/video-manager-impl.js

```diff
@@ -861,24 +861,44 @@ class VideoEntry {
     });
 
     // Listen to pause, play and user interaction events.
-    const unlistenInteraction = listen(mask, 'click', onInteraction.bind(this));
+    const unlisteners = [];
```

Comment:
`this.addListener_` (currently `this.addDragListener_` ) does the same job for drag/drop listeners. Maybe generalize it to take multiple unlistener lists?